README.md

```diff
@@ -35,8 +35,8 @@ First, write the following lines in your `.rootlogon.C`
 ```
 {
  gInterpreter->AddIncludePath("$LCIO");
- gSystem->Load("${LCIO}/lib/liblcio.so");
- gSystem->Load("${LCIO}/lib/liblcioDict.so");
+ gSystem->Load("liblcio.so");
+ gSystem->Load("liblcioDict.so");
 }
 ```
 and add $LCIO/lib to your LD_LIBRARY_PATH:
```

Comment:
```suggestion
and add $LCIO/lib and/or $LCIO/lib64 to your LD_LIBRARY_PATH:
```
(this potentially also implies some more changes to the following lines)

In order for ROOT to be able to locate the shared library in the `gSystem->Load` calls by name only, they have to be on `LD_LIBRARY_PATH`